packages/celotool/src/lib/attestation-service.ts

```diff
@@ -43,6 +44,10 @@ function helmParameters() {
     )}"`,
     `--set geth.validators="${fetchEnv(envVar.VALIDATORS)}"`,
     `--set domain.name=${fetchEnv(envVar.CLUSTER_DOMAIN_NAME)}`,
+    `--set global.postgresql.postgresqlDatabase=AttestationService`,
```

Comment:
Is a shared global db the long term plan or a stop gap?

Reply:
I'd say since this is only used for our testnets it is an acceptable solution for now? We currently use it only for a mutex around attestations so I think it's fine for now? Will change if requirements do